scripts/metavar.py

```diff
@@ -183,111 +266,147 @@ class VariableProperty(object):
     ['x:y']
     >>> VariableProperty('dimensions', list, check_fn_in=check_dimensions).valid_value('(w:x,y:z)')
     ['w:x', 'y:z']
+    >>> VariableProperty('dimensions', list, check_fn_in=check_dimensions).valid_value(['size(foo)'])
+    ['size(foo)']
     >>> VariableProperty('dimensions', list, check_fn_in=check_dimensions).valid_value('(w:x,x:y:z:q)', error=True) #doctest: +IGNORE_EXCEPTION_DETAIL
     Traceback (most recent call last):
     CCPPError: 'x:y:z:q' is an invalid dimension range
     >>> VariableProperty('dimensions', list, check_fn_in=check_dimensions).valid_value('(x:3y)', error=True) #doctest: +IGNORE_EXCEPTION_DETAIL
     Traceback (most recent call last):
     CCPPError: '3y' is not a valid Fortran identifier
+    >>> VariableProperty('local_name', str, check_fn_in=check_local_name).valid_value('foo')
+    'foo'
+    >>> VariableProperty('local_name', str, check_fn_in=check_local_name).valid_value('foo(bar)')
+    'foo(bar)'
+    >>> VariableProperty('local_name', str, check_fn_in=check_local_name).valid_value('q(:,:,index_of_water_vapor_specific_humidity)')
+    'q(:,:,index_of_water_vapor_specific_humidity)'
```

Comment:
The variables below have been removed in feature/capgen. Why are they back?

Reply:
git seems to have added this code back in apparently when I did the merge